### PR TITLE
Bump tar in notebook extension

### PR DESCRIPTION
**Pull request by kburtram**
Bump tar npm version in notebook extension.

### CODE REVIEW

File: extensions/notebook/package.json [hunk 1]
```diff
@@ -667,7 +667,7 @@
     "js-yaml": "^3.12.1",
     "node-fetch": "^2.6.1",
     "request": "^2.88.0",
-    "tar": "^6.0.1",
+    "tar": "^6.1.1",
```
Comment (Charles-Gagnon) [inline extensions/notebook/package.json:670]: Should bump this to 6.1.6 as well - especially since it's already set as that in the yarn.lock

Comment (kburtram) [reply extensions/notebook/package.json:670]: ok, will do.  I just grabbed the version mentioned in the alert.

**Comment by chlafreniere** [comment]
@lucyzhang929 let's make sure to test new python install after this package is bumped

**Comment by kburtram** [comment]
Bumped to the latest version of tar

**Pull request merged**